lib/matplotlib/ticker.py

```diff
@@ -242,6 +245,9 @@ def set_view_interval(self, vmin, vmax):
     def get_minpos(self):
         return self._minpos
 
+    def set_minpos(self, minpos=0):
```

Comment:
since this is new, should it be a [property](https://www.programiz.com/python-programming/property) and should get_minpos be deprecated....? Also, I acknowledge that's very likely out of scope.